thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/db/dao/AnomalyResultDAO.java

```diff
@@ -35,21 +39,39 @@
           + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
           + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
 
-  private static final String FIND_BY_TIME_AND_FUNCTION_ID = "SELECT r FROM AnomalyResult r WHERE r.function.id = :functionId "
-      + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
-      + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
+  private static final String FIND_BY_TIME_AND_FUNCTION_ID =
+      "SELECT r FROM AnomalyResult r WHERE r.function.id = :functionId "
+          + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
+          + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
 
-  private static final String FIND_BY_COLLECTION_TIME_AND_FUNCTION_ID = "SELECT r FROM AnomalyResult r WHERE r.function.collection = :collection "
-      + "AND r.function.id = :functionId "
-      + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
-      + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
+  private static final String FIND_BY_COLLECTION_TIME_AND_FUNCTION_ID =
+      "SELECT r FROM AnomalyResult r WHERE r.function.collection = :collection "
+          + "AND r.function.id = :functionId "
+          + "AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
+          + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc))";
 
   private static final String FIND_VALID_BY_TIME_EMAIL_ID =
       "SELECT r FROM EmailConfiguration d JOIN d.functions f JOIN f.anomalies r "
-      + "WHERE d.id = :emailId AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
-      + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc)) "
+          + "WHERE d.id = :emailId AND ((r.startTimeUtc >= :startTimeUtc AND r.startTimeUtc <= :endTimeUtc) "
+          + "OR (r.endTimeUtc >= :startTimeUtc AND r.endTimeUtc <= :endTimeUtc)) "
           + "AND r.dataMissing=:dataMissing";
 
+  private static final String COUNT_GROUP_BY_FUNCTION = "select count(r.id) as num, r.function.id,"
```

Comment:
minor nit for readability: can we keep this consistent in terms of what is capitalized? (all keywords like SELECT, GROUP BY etc have been capitalized in all other query definitions)